console/views/group.py

```diff
@@ -63,7 +63,7 @@ def post(self, request, *args, **kwargs):
         if len(note) > 2048:
             return Response(general_message(400, "node too long", "应用备注长度限制2048"), status=400)
 
-        data = group_service.create_app(self.tenant, self.response_region, app_name, note)
+        data = group_service.create_app(self.tenant, self.response_region, app_name, note, self.user.get_username())
```

Comment:
get_name ?

Reply:
The username is saved, and name will be used when getting details

Reply:
The application owner should save the user ID.

Reply:
why? The username is also unique.

Reply:
so, Associated with the username?

Reply:
Yes, username is the unique identity of the user